src/workflow/workflow.js

```diff
@@ -1066,6 +1084,139 @@ const runWorkflow = async (configFile = './src/workflow.json', dryRunMode) => {
   context.clean();
 };
 
+const mintKnownAddressesWorkflow = async (configFile = './src/workflow.json', dryRunMode) => {
```

Comment:
This seems the exact duplicate of runWorkflow. How about just passing a flag to runWorkflow (e.x. runWorkflow(..., withPresetAddress:boolean=true))
and branch the address section based on the flag value.

Reply:
Oh those flags.. :) I was afraid the code will become unreadable, but could give it a try tomorrow